modulefiles/gsi_hera.gnu.lua

```diff
@@ -1,13 +1,13 @@
 help([[
 ]])
 
-prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.6.0/envs/gsi-addon-dev-rocky8/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/scratch4/NCEPDEV/stmp/role.epic/spack-stack/spack-stack-1.6.0/envs/gsi-addon-dev-rocky8/install/modulefiles/Core")
 --Needed for openmpi build
-prepend_path("MODULEPATH", "/scratch1/NCEPDEV/jcsda/jedipara/spack-stack/modulefiles")
+--prepend_path("MODULEPATH", "/scratch1/NCEPDEV/jcsda/jedipara/spack-stack/modulefiles")
```

Comment:
I suggest deleting this line rather than commenting it.

Reply:
Agreed.  Good suggestion @DavidHuber-NOAA

Reply:
Done at [41f325f](https://github.com/NOAA-EMC/GSI/pull/833/commits/41f325f338baa8e2bd91cd119e790d2d0d7c2f76)